scripts/installscripts/buster-install-default.sh

```diff
@@ -811,19 +816,15 @@ install_main() {
     fi
 
     # prepare python3
-    ${apt_get} ${allow_downgrades} install python3 python3-dev python3-pip python3-setuptools python3-wheel python3-mutagen python3-gpiozero python3-spidev
-
-    # use python3 as default
-    sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1
```

Comment:
Does the script still work with Buster, if update-alternatives is removed?

Reply:
I have not checked that. My first pull request introduced another installer named `bullseye-install-default.sh` but I was asked to make the changes to the buster script.
What I have checked though, is that these parameters are part of the reasons why the script fails in bullseye, which is why I removed them.

But to be honest: I do not see why backwards compatibility is a thing: This script is meant to be part of a guide, which instructs the user to start with a fresh bullseye Pi-Image. I do not see a point in adding extra efforts to support any previous Debian distribution as well.

Reply:
I think buster support shouldn't be breaked yet, as it is an LTS Version since June 30th, 2024 and propably quite a few installations are still active that just want to update.
But i checked the python calls in the source code and as far as i can tell they are all using `python3`, so removing this shouldn't have any impact.